### PR TITLE
Cleanup in functional tests

### PR DESCRIPTION
**Pull request by ahadas**
**What this PR does / why we need it**:
This PR removes two functions from `tests/utils.go` and minimize the generation of random names within the tests.

**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: tests/migration_test.go [hunk 1]
```diff
@@ -224,7 +224,7 @@ var _ = Describe("[rfe_id:393][crit:high[vendor:cnv-qe@redhat.com][level:system]
 	runMigrationAndExpectCompletion := func(migration *v1.VirtualMachineInstanceMigration, timeout int) string {
 		By("Starting a Migration")
 		Eventually(func() error {
-			_, err := virtClient.VirtualMachineInstanceMigration(migration.Namespace).Create(migration)
+			migration, err = virtClient.VirtualMachineInstanceMigration(migration.Namespace).Create(migration)
```
Comment (vladikr) [inline tests/migration_test.go]: I think you need to do the same everywhere we create a migration object, otherwise, migration name will remain empty...

Comment (ahadas) [reply tests/migration_test.go]: Done

**Pull request merged**